packit_service/worker/events/event.py

```diff
@@ -119,8 +119,16 @@ def from_event_dict(cls, event: dict):
         time = event.get("task_accepted_time")
         task_accepted_time = datetime.fromtimestamp(time, timezone.utc) if time else None
 
-        build_targets_override = event.get("build_targets_override")
-        tests_targets_override = event.get("tests_targets_override")
+        build_targets_override = (
+            {(target, identifier_) for [target, identifier_] in event.get("build_targets_override")}
+            if event.get("build_targets_override")
+            else set()
```

Comment:
just to be sure, I also remember some discussions about the empty set vs None for *targets_override, is it ok to have empty set here instead of None?

Reply:
I think it is, however I want to test this a bit deeper in the staging instance.